### PR TITLE
add minutes from team container for 2023-01-23

### PR DESCRIPTION
**Pull request by fkr**
Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

File: container/20230123.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* AI @garloff: Open an issue to make sure it is present for next weeks sprint planning
+
+
+#### Loadbalancer discussions continued
+(please dive into the issues, since they cover everything that was said in the meeting)
+* Want to get client IP: [#312](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/312)
+	[#169](https://github.com/SovereignCloudStack/standards/pull/169) [#227](https://github.com/SovereignCloudStack/issues/issues/227)
+	* Proxy protocol investigation progress [#250](https://github.com/SovereignCloudStack/issues/issues/250)
+		* How do we avoid local traffic to be broken? [#184](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/184)
+	* L3 loadbalancer options (#251)[https://github.com/SovereignCloudStack/issues/issues/251]
```
Comment (o-otte) [inline container/20230123.md:44]:
```suggestion
	* L3 loadbalancer options [#251](https://github.com/SovereignCloudStack/issues/issues/251)
```
The Link formatting seems off

**Review by o-otte — COMMENTED** [review]
One link is off, the rest lgtm

File: container/20230123.md (hunk 1 above)
Comment (fkr) [reply container/20230123.md]: thanks @o-otte, well spotted.

**Review by o-otte — APPROVED** [review]
lgtm

**Pull request merged**